isis/tests/FunctionalTestsFindfeatures.cpp

```diff
@@ -248,3 +248,34 @@ TEST_F(ThreeImageNetwork, FunctionalTestFindfeaturesErrorNoMatch) {
     EXPECT_THAT(e.what(), HasSubstr("**USER ERROR** No control points found!"));
   }
 }
+
+TEST_F(ThreeImageNetwork, FunctionalTestFindfeaturesTiled) {
```

Comment:
This test is not ideal.  We need to find a way to construct an image other than "noise" so that features still match after geometric correction.

Reply:
I agree, but this is a general problem with the findfeatures testing not being very helpful in the first place. All of the tests are effectively testing the OpenCV feature matching which has caused issues when we make small tweaks to things.

I think this is acceptable for now.